update-SensorDict.py

```diff
@@ -0,0 +1,71 @@
+import requests
+import pandas as pd
+# Maybe you can do this with the csv library
+# But Pandas is way easier to use
+
+file_id = "195Y2cf9C2mrA6QCLuLJv3NHsCqTLi-3Wd07q4T5rXuM"
+output_file = "SavedGoogleDrive.csv"
+# Index of the header row:
+ignore_rows = 5
+
+# This only gets the FIRST SHEET in the document
+# If there is more than one sheet, make sure the first sheet is correct
+response = requests.get("https://docs.google.com/spreadsheet/ccc?key="
+                        + file_id + "&output=csv")
+# Check that status code = 200
+response.raise_for_status()
+
+# The CSV spreadsheet is requests.content
+# Turn it into file:
+savefile = open(output_file, "wb")
+savefile.write(response.content)
+savefile.close()
+
+# Read file as dataframe.
+content_df = pd.read_csv(output_file, sep=",", header=ignore_rows, dtype="str")
+
+# Drop unwanted columns:
+content_df = content_df.drop(["TRANSMITTING MODULE", "DESCRIPTION / SENSOR",
+                              "AddMethd"], axis="columns")
```

Comment:
Is this supposed to be `AddMethd`?

Reply:
Yes, it is how it is formatted in the sheet.